### PR TITLE
handbook update per March 23 meeting

### PR DESCRIPTION
**Pull request by sgoggins**

### CODE REVIEW

**Comment by sgoggins** [comment]
@klumb / @GeorgLink  / @jaskiratsingh2000 : I added the "rules" we discussed in the March 23rd community meeting about metrics maintenance.  That's all this is.  Feel free to make suggestions. I'm sure I didn't fully follow the style, but I tried.

**Comment by GeorgLink** [comment]
How about we add this to the metrics release page to keep the page count down and keep relevant things together?

Seems related to: https://handbook.chaoss.community/community-handbook/community-initiatives/metrics#when-should-a-previously-released-metric-be-returned-to-community-review

**Comment by sgoggins** [comment]
> How about we add this to the metrics release page to keep the page count down and keep relevant things together?
> 
> Seems related to: https://handbook.chaoss.community/community-handbook/community-initiatives/metrics#when-should-a-previously-released-metric-be-returned-to-community-review

Will do!

**Comment by sgoggins** [comment]
> How about we add this to the metrics release page to keep the page count down and keep relevant things together?
> 
> Seems related to: https://handbook.chaoss.community/community-handbook/community-initiatives/metrics#when-should-a-previously-released-metric-be-returned-to-community-review

Fixed. Great suggestion!  Thanks, @Georg!

**Comment by sgoggins** [comment]
Also, my feeling will not be hurt if anyone edits the fork to make direct changes if they want. I am also happy to make the changes. Its open on my desktop until merged.

File: community-initiatives/metrics/chaoss-metric-release.md [hunk 1]
```diff
@@ -107,3 +107,24 @@ After creating the merged document, add the front-matter
   * Bookmarks handling: `Retain bookmarks`
   * Table of contents: `Don't generate`
 
+---
+description: Maintaining, Reviewing and Updating Released Metrics
+---
+
```
Comment (GeorgLink) [inline community-initiatives/metrics/chaoss-metric-release.md:113]:
```suggestion
```

Comment (GeorgLink) [reply community-initiatives/metrics/chaoss-metric-release.md:113]: Remove superfluous artifact from copying the text here.

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @sgoggins 

I just request removing the description that was left over from when this was it's stand-alone page.

**Review by GeorgLink — APPROVED** [review]

**Comment by GeorgLink** [comment]
Thanks again @sgoggins!

The edit is now live: https://handbook.chaoss.community/community-handbook/community-initiatives/metrics/chaoss-metric-release#revising-existing-metrics

**Pull request merged**